inc/deployfile.class.php

```diff
@@ -226,7 +226,7 @@ static function displayList(PluginFusioninventoryDeployOrder $order, $datas, $ra
             echo "<a title='".__('p2p', 'fusioninventory').", "
             .__("retention", 'fusioninventory')." : ".
                $file_p2p_retention_duration." ".
-               __("days", 'fusioninventory')."' class='more'>";
+               __("Minute(s)", 'fusioninventory')."' class='more'>";
```

Comment:
Are you sure it's in minutes ? I remember it was in days (but perhaps it's changed in agent)